### ISSUE TITLE
The method setSvgResource() is no use

### ISSUE THREAD

**Issue by nbalichaoq** [geftimov/android-pathview#37]
when I invoke this method setSvgResource() to change the resourse of SVG, the SVG animator is still the playing as the old one

**Comment by geftimov** [comment]
That is true , should be easy fix stopping the animator every time we change the SVG. 
Unfortently I don't have time to do it. If you are interested in contributing , you can make a PR for it.
